extensions/AzureAISearch/AzureAISearch.FunctionalTests/AzureAISearch.FunctionalTests.csproj

```diff
@@ -9,6 +9,7 @@
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
         <IsPackable>false</IsPackable>
+        <UserSecretsId>31bdda67-28ce-4ba2-b3e7-0eca5d687404</UserSecretsId>
```

Comment:
We actually removed this a while back, in favor of using `appsettings.development.json`. 

Also, we're pushing for a more secure authentication, using Managed Identities, so I would avoid reintroducing this approach, even if that's for services outsize Azure.
 
(Btw, using a different secret store ID for each project would mean having to store secrets multiple times)